pulseapi/creators/migrations/0001_initial.py

```diff
@@ -21,7 +18,6 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.CharField(max_length=140)),
-                ('user', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='as_creator', to=settings.AUTH_USER_MODEL)),
```

Comment:
that would be my suggestion too, yes. It's time to wipe the compound migrations we've been accumulating (except for issues, which is one step and has data) and rebuild everything as single step migrations.